src/components/AssetsFilters/AssetsFilters.scss

```diff
@@ -1,4 +1,9 @@
-ul.filter-set {
-  list-style: none;
-  padding-left: $spacer;
+@import "~bootstrap/scss/utilities/_spacing.scss";
+
+.filter-heading {
+	@extend .pt-3;
```

Comment:
I just want to double check: the .pt-3 classname comes from the Bootstrap spacing utilities, right?

Reply:
Yep - https://getbootstrap.com/docs/4.0/utilities/spacing/ I found it from the PR Dennis had in Paragon with table headings.